partner_multi_relation/README.rst

```diff
@@ -0,0 +1,104 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=================
+Partner Relations
+=================
+
+This module aims to provide generic means to model relations between partners.
+
+Examples would be 'is sibling of' or 'is friend of', but also 'has contract X
+with' or 'is assistant of'. This way, you can encode your knowledge about your
+partners directly in your partner list.
+
+Installation
+============
+
+Configuration
+=============
+
+Usage
+=====
+
+Before being able to use relations, you'll have define some first. Do that in
+Sales / Configuration / Address Book / Partner relations. Here, you need to
+name both sides of the relation: To have an assistant-relation, you would name
+one side 'is assistant of' and the other side 'has assistant'. This relation
+only makes sense between people, so you would choose 'Person' for both partner
+types. For the relation 'is a competitor of', both sides would be companies,
+while the relation 'has worked for' should have persons on the left side and
+companies on the right side. If you leave this field empty, the relation is
+applicable to all types of partners.
+
+If you use categories to further specify the type of partners, you could for
+example enforce that the 'is member of' relation can only have companies with
+label 'Organization' on the left side.
+
+Now open a partner and choose relations as appropriate in the 'Relations' tab.
+
+Searching partners with relations
+---------------------------------
+
+Searching for relations is integrated transparently into the partner search
+form. To find all assistants in your database, fill in 'is assistant of' and
```

Comment:
Actually is searching both sides of the relation. Is it right?

Reply:
yes it does, I just tested it.

Reply:
I meant if you try "To find all assistants in your database, fill in 'is assistant of' and autocomplete will propose to search for partners having this relation" you will not get all the assistants --> You will get all the assistants & all the contacts that "Has assistant".
The option of search just one side of a relation could be an improvement for a new release in the future.

Reply:
ok, got it